### PR TITLE
Allow bind addresses for web and admin servers to be configurable

### PR DESCRIPTION
**Pull request by luismarques**

### CODE REVIEW

File: source/vibenews/vibenews.d [hunk 1]
```diff
@@ -27,11 +27,13 @@ class VibeNewsSettings {
...
 	ushort nntpSSLPort = 563;
 	deprecated @property ref inout(ushort) nntpSslPort() inout { return nntpSSLPort; }
-	ushort webPort = 8009;
+	ushort webPort = 80;
 	ushort adminPort = 9009;
 	string databaseName = "vibenews";
 	string sslCertFile;
 	string sslKeyFile;
+    string[] webBindAddresses;
+    string[] adminBindAddresses;
```
Comment (s-ludwig) [inline source/vibenews/vibenews.d]: Indentation is off here (spaces vs. tabs).

**Comment by s-ludwig** [comment]
LGTM otherwise!

**Comment by luismarques** [comment]
I fixed the whitespace issue and squashed everything into a single commit (was that a bad idea?).

**Comment by luismarques** [comment]
This is still open. Is anything missing?

**Comment by s-ludwig** [comment]
Ah, no, looks good. Sorry, lost track of this.

**Pull request merged**